client/post-editor/editor-categories-tags/accordion.jsx

```diff
@@ -36,10 +36,6 @@ export class EditorCategoriesTagsAccordion extends Component {
 		defaultCategory: PropTypes.object,
```

Comment:
Does this component need to accept a `post` prop anymore? Only place it appears to be used is in returning `null` early from `render`.